lib/utils/testLauncherHelper.js

```diff
@@ -268,6 +208,114 @@ async function launchChild(cmdArgv, ownArgv, options, runMode) {
 	);
 }
 
+/**
+ * Save child output to the file
+ * @param {ChildProcess} child
+ * @param {Buffer} chunk
+ * @param {WritableStream?} logStream
```

Comment:
what is the point of making logStream optional if without it function will not work? I think it should be marked as required for proper IDE support

Reply:
This function exists for code organization only - i needed to make the calling function shorter and hide the optionality of the logStream.

But you are right it should have been marked as WriteableStream|null instead